virttest/virt_vm.py

```diff
@@ -1216,6 +1216,9 @@ def print_guest_network_info():
                 if serial:
                     break
                 raise
+            except remote.LoginProcessTerminatedError as err:
+                error = err
```

Comment:
I notice the output log comes from serial console, so suggest add conditional here:
if serial:
     break

Moreover, have you any chance to test the change ? please upload some test results if you have.